### PR TITLE
Added mitigation for package version mismatch

### PR DESCRIPTION
**Pull request by marknoble**
While completing this step with VS 2022 and SQL Server 2022 Dev edition, I encountered this issue.

After trying a suggestion found here: https://johnkilleenblog.wordpress.com/2019/10/08/ssis-script-component-mismatch-error/#:~:text=Project%20Properties%20%E2%80%93%20Configuration%20Properties%20%E2%80%93%20General

I was able to resolve the error and proceed with the Tutorial.

I have added those steps as an Important block, for people who encounter the error, but perhaps the issue could be avoided by properly setting this value proactively at an earlier step in the Tutorial.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@chugugrace 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md [hunk 1]
```diff
@@ -62,7 +62,18 @@ In this task, you:
   
 7.  Select **Next**.  
   
-8.  On the Select Packages page, you should see all six packages from the SSIS Tutorial. In the **Packages** list, select **Lesson 6.dtsx**, then select **Next**.  
+8.  On the Select Packages page, you should see all six packages from the SSIS Tutorial. In the **Packages** list, select **Lesson 6.dtsx**, then select **Next**.
+
+> [!IMPORTANT]
+> If the above step results in an error that _"One or more selected packages are not ready. Review the Status column for more information."_ with an Status message including _"The version or pipeline version or both for the specified component is higher than the current version. This package was probably created on a new version of DTS or the component than is installed on the current PC."_ Try changing the Project Properties in Visual Studio as follows:
```
Comment (rwestMSFT) [inline docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md]:
```suggestion
> The previous step may result in an error stating that "One or more selected packages are not ready. Review the Status column for more information." with a status message including "The version or pipeline version or both for the specified component is higher than the current version. This package was probably created on a new version of DTS or the component than is installed on the current PC." To resolve this error, change the Project Properties in Visual Studio as follows:
```

File: docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md [hunk 2]
```diff
@@ -62,7 +62,18 @@ In this task, you:
...
   
-8.  On the Select Packages page, you should see all six packages from the SSIS Tutorial. In the **Packages** list, select **Lesson 6.dtsx**, then select **Next**.  
+8.  On the Select Packages page, you should see all six packages from the SSIS Tutorial. In the **Packages** list, select **Lesson 6.dtsx**, then select **Next**.
+
+> [!IMPORTANT]
+> If the above step results in an error that _"One or more selected packages are not ready. Review the Status column for more information."_ with an Status message including _"The version or pipeline version or both for the specified component is higher than the current version. This package was probably created on a new version of DTS or the component than is installed on the current PC."_ Try changing the Project Properties in Visual Studio as follows:
+> 
+> 1.  Right-click the **SSIS Tutorial (package deployment)** project and select **Properties**.
+> 
+> 2.  Under **Configuration Properties** select **General**.
```
Comment (rwestMSFT) [inline docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md]:
```suggestion
> 2.  Under **Configuration Properties**, select **General**.
```

File: docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md [hunk 3]
```diff
@@ -62,7 +62,18 @@ In this task, you:
...
+> 
+> 3.  Set the **Deployment Target Version** to an older version, for example changing from **SQL Server 2022** to **SQL Server 2019**, then select **OK**.
```
Comment (rwestMSFT) [inline docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md]:
```suggestion
> 3.  Set the **Deployment Target Version** to an older version. For example, change from **SQL Server 2022** to **SQL Server 2019**, then select **OK**.
```

File: docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md [hunk 4]
```diff
@@ -62,7 +62,18 @@ In this task, you:
...
+> 
+> 4.  Retry the above step.
```
Comment (rwestMSFT) [inline docs/integration-services/lesson-6-4-deploying-the-lesson-6-package.md]:
```suggestion
> 4.  Retry the previous step.
```

**Review by rwestMSFT — CHANGES_REQUESTED** [review]
Thanks for this contribution. I've made some minor recommendations for style.

**Comment by rwestMSFT** [comment]
@marknoble Thanks for going to this trouble. I had already incorporated your other two PRs yesterday and forgot to include this one, so that's on me. I'll get this sorted out ASAP as another internal PR, so I can squash the commits.

Thank you for taking the time to contribute.

**Comment by rwestMSFT** [comment]
@marknoble The changes from this PR have been incorporated into an internal PR, and will be published in the next day or so. Once the internal PR is merged, this PR will automatically close. Thank you again for contributing to Microsoft Docs.

**Pull request closed without merging**